app/serializers/order_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class OrderSerializer < ActiveModel::Serializer
+  attributes :id, :name, :starts, :ends, :boxfill, :pickup, :is_open, :is_boxfill
+
+  def is_open
+    object.open?
+  end
+
+  def is_boxfill
+    !!object.boxfill?
```

Comment:
no, you're right, thanks!

Reply:
Ah, now I see why: this value isn't expected to be `null` in the API output, and it can be.
I'll change this in the model, so that it always returns a boolean value (which makes sense for a `?` method).